File: eBotanika.Services/Database/Dostava.cs
namespace eBotanika.Services.Database
{
    public partial class Dostava
    {
        public int? DostavaID { get; set; }
        public DateTime? DatumSlanja { get; set; }
        public DateTime? DatumPreuzimanja { get; set; }
        public string? TrenutnaLokacija { get; set; }
        public string? Napomena { get; set; }
        public int? RezervacijaID { get; set; }
        public Rezervacije? Rezervacije { get; set; }
    }
}

File: eBotanika.Mobile/lib/models/dostava.dart
class Dostava {
  int dostavaID;
  DateTime datumSlanja;
  DateTime datumPreuzimanja;
  String trenutnaLokacija;
  String napomena;
  int rezervacijaID;

  Dostava({
    required this.dostavaID,
    required this.datumSlanja,
    required this.datumPreuzimanja,
    required this.trenutnaLokacija,
    required this.napomena,
    required this.rezervacijaID,
  });

  factory Dostava.fromJson(Map<String, dynamic> json) {
    return Dostava(
        dostavaID: int.parse(json["dostavaID"].toString()),
        datumSlanja: json["datumSlanja"],
        datumPreuzimanja: json["datumPreuzimanja"],
        trenutnaLokacija: json["trenutnaLokacija"],
        napomena: json["napomena"],
        rezervacijaID: int.parse(json["rezervacijaID"].toString()));
  }

  Map<String, dynamic> toJson() => {
        "dostavaID": dostavaID,
        "datumSlanja": datumSlanja,
        "datumPreuzimanja": datumPreuzimanja,
        "trenutnaLokacija": trenutnaLokacija,
        "napomena": napomena,
        "rezervacijaID": rezervacijaID
      };
}